lib/y2_module_guitest.pm

```diff
@@ -60,7 +62,8 @@ sub launch_yast2_module_x11 {
     diag 'assuming root-auth-dialog, typing password';
     type_password;
     send_key 'ret';
-    assert_screen $args{target_match}, $args{match_timeout};
+    wait_still_screen 2, 2;
+    is_sle('>=15-SP4') ? apply_workaround_poo124652($args{target_match}, $args{match_timeout}) : assert_screen $args{target_match}, $args{match_timeout};
```

Comment:
we need to pass a new argument to do a condition here, so we don't need to do the condition by product inside this method but outside in the call and here just `apply_workaround_* if args{apply_workaround}`.